src/kixi/heimdall/member.clj

```diff
@@ -4,18 +4,26 @@
 
 (defn retrieve-groups-ids
   [session id]
-  (db/select session :members_by_user [:group-id] {:user-id id}))
+  (map :group-id (db/select session :members_by_user [:group-id] {:user-id id})))
 
 (defn add-user-to-group
-  [session user-id group-id role]
+  [session user-id group-id]
   {:pre [user-id group-id]}
   (db/insert! session :members_by_group
               {:id (uuid/random)
                :user-id user-id
-               :group-id group-id
-               :role role})
+               :group-id group-id})
   (db/insert! session :members_by_user
```

Comment:
cassandra doesn't have transactions, we might have to hand-roll something if we really want it